SystemInformer/mainwnd.c

```diff
@@ -284,46 +284,161 @@ PPH_STRING PhMwpInitializeWindowTitle(
     )
 {
     PH_STRING_BUILDER stringBuilder;
-    PPH_STRING currentUserName;
+    PPH_STRING myUserAndDomain;
 
     if (!PhEnableWindowText)
     {
         PhApplicationName = L" ";
         return NULL;
     }
 
-    PhInitializeStringBuilder(&stringBuilder, 100);
-    PhAppendStringBuilder2(&stringBuilder, PhApplicationName);
-
-    if (currentUserName = PhGetSidFullName(PhGetOwnTokenAttributes().TokenSid, TRUE, NULL))
+    if (!PhGetIntegerSetting(L"EnableVerboseDeveloperWindowTitle"))
```

Comment:
Would it be cleaner to move two into their own functions and not invert the statement?:

```cpp
if (PhGetIntegerSetting(L"EnableVerboseWindowTitle"))
    return PhMwpVerboseWindowTitle();
else
    return PhMwpStandardWindowTitle();
```